app/lib/search/index_simple.dart

```diff
@@ -609,7 +609,8 @@ class TokenMatch {
 
 class TokenIndex {
   final Map<String, String> _textHashes = <String, String>{};
-  final Map<String, Set<String>> _inverseIds = <String, Set<String>>{};
+  final Map<String, Map<String, double>> _inverseIds =
```

Comment:
You don't need to duplicate the generic type, final _inverseIds = <foo, bar>{} will deduce the correct type.